root/user/components/UserInlineList.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/license/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {l} from '../../static/scripts/common/i18n';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+type Props = {|
+  +editors: EditorT,
+|};
+
+const UserInlineList = ({editors}: Props) => {
```

Comment:
Opening bracket is not appropriate here as it should be used to enclose JavaScript expression whereas `<p>` is a JSX element. You can replace it with opening parenthesis if you like. (And update closing bracket on line 32 accordingly.)

Reply:
Change `{` with `(` , right?